Libraries/Components/ScrollResponder.js

```diff
@@ -471,6 +472,15 @@ var ScrollResponderMixin = {
    * @platform ios
    */
   scrollResponderFlashScrollIndicators: function() {
+    if (Platform.OS === 'android') {
```

Comment:
But there should not be any native changes required on iOS, it should just work when using dispatchViewManagerCommand